apps/web/styles/globals.css

```diff
@@ -43,78 +60,119 @@ body {
   line-height: 1.15;
 }
 
-/* TODO: add Inter Font */
-
 h1 {
```

Comment:
you can also customize components like this:
![image](https://github.com/1Hive/gardens-v2/assets/13890618/d5459fe3-2926-40b3-82df-c4b602af3208)
https://daisyui.com/docs/customize/

Reply:
Yes that an option we can adopt maybe for component level classNames ?
In this case we just customize a bit some basic html tags